packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/multi-party.js

```diff
@@ -0,0 +1,166 @@
+/*!
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '@ciscospark/plugin-phone';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import sinon from '@ciscospark/test-helper-sinon';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import handleErrorEvent from '../lib/handle-error-event';
+import {browserOnly, maxWaitForEvent} from '@ciscospark/test-helper-mocha';
+
+if (process.env.NODE_ENV !== `test`) {
+  throw new Error(`Cannot run the plugin-phone test suite without NODE_ENV === "test"`);
+}
+
+browserOnly(describe)(`plugin-phone`, function() {
+  this.timeout(60000);
+
+  describe(`Phone`, () => {
+    let mccoy, spock;
+    before(`create users and register`, () => testUsers.create({count: 2})
+      .then((users) => {
+        [mccoy, spock] = users;
+        spock.spark = new CiscoSpark({
+          credentials: {
+            authorization: spock.token
+          }
+        });
+
+        mccoy.spark = new CiscoSpark({
+          credentials: {
+            authorization: mccoy.token
+          }
+        });
+
+        return Promise.all([
+          spock.spark.phone.register(),
+          mccoy.spark.phone.register()
+        ]);
+      }));
+
+    let ringMccoy;
+
+    beforeEach(() => {
+      ringMccoy = sinon.spy();
+      mccoy.spark.phone.on(`call:incoming`, ringMccoy);
+    });
+
+    beforeEach(() => {
+      spock.spark.config.phone.enableExperimentalGroupCallingSupport = true;
+      mccoy.spark.config.phone.enableExperimentalGroupCallingSupport = true;
+    });
+
+    afterEach(() => {
```

Comment:
Is there a reason why we are unsetting these after each test?

Reply:
It's good practice in case these users get used elsewhere. I've made attempts to reuse test users in separate files in the past, so just in case we try that again, we want to make sure they're in their original state.

Reply:
Gotcha, thanks!